Makefile

```diff
@@ -127,19 +159,14 @@ push:
 .PHONY: push
 
 # Rebuild .proto files
-generate: $(BUF) $(PROTOC)
+generate:
 	go generate ./...
-	$(BUF) generate
+	buf generate
 .PHONY: generate
 
-# Lint .proto files
-lint-protos: $(BUF) $(PROTOC)
-	$(BUF) lint
-.PHONY: lint-protos
-
 # Verify if files built from .proto are up to date.
-check: generate
-	@git add -N $(PROTOC_BUILD_DIRS)
-	@git diff --name-only --diff-filter=AM --exit-code $(PROTOC_BUILD_DIRS) \
+test-generate: generate
+	@git add -N .
```

Comment:
This line will add **all** _untracked_ files a dev has to _tracked_ files, including any scratch files etc one might have locally. It will mess with ppl's workspace :(

At least leave a warning in a comment...